simulate-event.js

```diff
@@ -340,7 +339,7 @@ module.exports = function (element, type, options) {
   options = extend({
     bubbles: true,
     cancelable: true
-  }, eventOptions[eventType](element, type, options), options)
+  }, eventOptions[eventType](type, options), options)
```

Comment:
I was just looking at this code, if `new Constructor` previously succeeds doesn't that mean all the options are ignored currently? Do we need to merge these options earlier in the generate function?